lerna.json

```diff
@@ -3,5 +3,10 @@
     "packages/*"
   ],
   "version": "independent",
-  "npmClient": "yarn"
+  "npmClient": "yarn",
+  "command": {
+    "version": {
+      "message": "chore(release): publish"
```

Comment:
This is the Git commit message that `lerna version` will use.

Since we have `version: independent`, it'll add info to the message's body with each packages' version bump.

https://github.com/lerna/lerna/tree/master/commands/version#--message-msg